packages/devextreme/js/localization/messages/ar.json

```diff
@@ -182,6 +182,7 @@
         "dxDataGrid-ariaExpand": "توسيع",
         "dxDataGrid-ariaCollapsedRow": "Collapsed row",
         "dxDataGrid-ariaExpandedRow": "Expanded row",
+        "dxDataGrid-ariaExpandableInstruction": "Use Enter key to toggle expand and collapse a focused node",
```

Comment:
We do not need an instruction for DataGrid because it supports standard hotkeys.